src/Core/Checkout/Customer/SalesChannel/RegisterRoute.php

```diff
@@ -316,6 +317,10 @@ private function validateRegistrationData(DataBag $data, bool $isGuest, SalesCha
             ));
         }
 
+        if ($this->systemConfigService->get('core.loginRegistration.requireDataProtectionCheckbox', $context->getSalesChannel()->getId())) {
+            $definition->add('dataProtectionInformation', new NotBlank());
```

Comment:
```suggestion
            $definition->add('acceptedDataProtection', new NotBlank());
```

Maybe 🤔 . @mitelg or what do you think?

Reply:
yeah, it should have some "boolean" name like `isThisAndThat` or `thatIsMade`